src/components/Image.js

```diff
@@ -33,15 +43,22 @@ export const Image = ({
     // -> https://juliangaramendy.dev/use-promise-subscription/
     let mounted = true;
 
-    source.uri
-      ? CacheManager.get(source.uri)
-          .getPath()
-          .then((path) => mounted && setUri(path))
-          .catch((err) => console.warn('An error occurred with cache management for an image', err))
-      : mounted && setUri(source);
+    if (refreshInterval === undefined) {
+      source.uri
+        ? CacheManager.get(source.uri)
+            .getPath()
+            .then((path) => mounted && setUri(path))
+            .catch((err) =>
+              console.warn('An error occurred with cache management for an image', err)
+            )
+        : mounted && setUri(source);
+    } else {
+      // add an artificial query param to the end of the url to trigger a rerender and refetch
+      mounted && setUri(addQueryParam(source.uri ?? source, `svaRefreshCount=${count}`));
+    }
 
     return () => (mounted = false);
-  }, [source, setUri]);
+  }, [count, refreshInterval, source, setUri]);
```

Comment:
i find the variable name `count` too general. one do not know a quick read, what is counted. what about `svaRefreshCount`? your already made your thoughts about a naming :)

Reply:
i set it to `refreshCount`. adding the `sva` prefix feels unnecessary for the internal variable. i only added it to the query param name to make it considerably less likely that that param actually has any influence

Reply:
good choice